pkg/scheduler/pool_external.go

```diff
@@ -410,6 +415,10 @@ func (wpc *ExternalWorkerPoolController) getWorkerEnvironment(workerId, machineI
 			Name:  "NETWORK_PREFIX",
 			Value: machineId,
 		},
+		{
```

Comment:
This may or may not be required, but I thought having the env var could useful worker side. It's also saved on the worker state itself, so this may be redundant.